release_package.rb

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+
+options = {
+  :jenkins_job_id => 'lastSuccessfulBuild'
+}
+
+OptionParser.new do |opts|
+  opts.banner = "Usage: release_package.rb [options]"
+
+  opts.on("-p", "--project [PROJECT]", "Project to build an RPM for") do |project|
+    options[:project] = project
+  end
+
+  opts.on("-r", "--releaser REQUIRED", "Tito releaser to release for") do |releaser|
+    options[:releaser] = releaser
+  end
+
+  opts.on("--jenkins-job [JENKINS_JOB]", "Name of Jenkins job to pull source file from") do |job|
+    options[:jenkins_job] = job
+  end
+
+  opts.on("--jenkins-job-id [JENKINS_JOB_ID]", "ID of Jenkins job to pull source from (default: lastSuccessfulBuild") do |id|
+    options[:jenkins_job_id] = id unless id.nil?
+  end
+
+  opts.on("--source_dir [SOURCE DIR]", "Directory which contains the source of the package being built (you can create it with gem unpack)") do |source_dir|
+    options[:source_dir] = source_dir unless source_dir.nil?
+  end
+end.parse!
+
+raise OptionParser::MissingArgument, "--project" if options[:project].nil?
+raise OptionParser::MissingArgument, "--releaser" if options[:releaser].nil?
+
+def log(msg)
+  puts
+  puts "== #{msg} =="
+end
+
+log("Initializing git annex")
+puts `git-annex init`
+
+log("Setting up source")
+if options[:project] == 'rubygem-katello' || options[:project] == 'katello-installer'
```

Comment:
These packages don't exist in this repo so I'd leave it out. If they are indeed needed then I think it should be fixed in ```setup_sources.sh``` since that's also used directly by users.